src/data/talks.json

```diff
@@ -0,0 +1,172 @@
+[
+  {
+    "id": 1,
+    "title": "Isomorphic React without Node",
+    "subtitle": "Ben Ilegbodu from Eventbrite",
+    "description": "Listen to <span>Ben Ilegbodu</span> from <span>Eventbrite</span> as he takes you through some really cool stuff that pertains to isomorphic react without using Node.",
```

Comment:
For this MVP we are not going to support rich contents. Use plain text.

Reply:
ok i remove this but the content don't will have the same styles in the description